src/shared/misc.c

```diff
@@ -553,8 +553,14 @@ struct notify notify_parse(const char *applet, const char *notify_string)
 			struct sockaddr header;
 			struct sockaddr_un unix;
 		} addr = { .unix = { .sun_family = AF_UNIX } };
-		int written = snprintf(addr.unix.sun_path, sizeof(addr.unix.sun_path), "%s/supervise-%s.sock", rc_svcdir(), applet);
-
+		int written;
+		const char *opts = strchr(notify_string, ':');
```

Comment:
This is inside a `strcmp(notify_string, "socket") == 0` branch, so none of this will ever be triggered.